embulk-output-redshift/src/main/java/org/embulk/output/RedshiftOutputPlugin.java

```diff
@@ -83,6 +83,14 @@
         @Config("ssl")
         @ConfigDefault("\"disable\"")
         public Ssl getSsl();
+
+        @Config("copy_unload_iam_role_name")
+        @ConfigDefault("\"\"")
```

Comment:
The default value should be null rather than empty string because the value won't be concatenated as 's3_key_prefix'.
(The default value for 'iam_user_name' also be null and I'll correct in future.)

Reply:
I wrote it because, personally I like default value if it is able to set rather than null, and same as other args.
This arg is used only set, so null is better to show intention it is optional value.
I fix it.